tests/users.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "username": "cramakri",
```

Comment:
Isn't the whole point to use robot users?

Reply:
> see my comment in renku-demo -- I would prefer to leave the users there untouched and instead inject users.json into the test container here via a configmap

There is coupling with `renku-demo`. The default has to be the same here and there.